src/components/pages/ReportForm/Steps/Step2.vue

```diff
@@ -0,0 +1,79 @@
+<template lang="pug">
+.step2
+  h3 {{ $t('step2.title') }}
+
+  label Describa el error encontrado
+  textarea(name="desc", 
+        v-model="issue.desc")
+
+  label Prioridad
+  select(name="priority", v-model="issue.priority")
+    option(value=1) BAJA
+    option(value=2) MEDIA
+    option(value=3) ALTA
+
+  label Criticidad
+  select(name="criticity", v-model="issue.criticity")
+    option(value=1) BAJA
+    option(value=2) MEDIA
+    option(value=3) ALTA
+
+  button(@click="addIssue") Agregar
+
+  .issueList(v-if="report.issues.length > 0")
+    h3 Issues cargados
+    hr
+    .issueDetail(v-for="i in report.issues")
+
+      p {{ i.desc }}
+      p 
+        b Prioridad
+        | {{ i.priority }}
+      p 
+        b Criticidad
+        | {{ i.criticity }}
+      ul
+        a(href='#', @click="editIssue(i)") Editar
+        a(href='#', @click="removeIssue(i)") Eliminar
+      hr
+</template>
+
+<script>
+import { mapGetters } from 'vuex'
+
+export default {
+  computed: {
+    ...mapGetters({
+      report: 'report',
+      issue: 'issue'
+    })
+  },
+  mounted () {
+    this.$bus.$on('validateStep2', this.validate)
+  },
+  methods: {
+    validate () {
+      if (this.report.issues.length > 0) {
+        this.errors.items = []
```

Comment:
Keep in mind that by doing this, you are overwriting the errors object created by vee-validate. I'd rather use another object/array than doing this